### ISSUE TITLE
Transformer 'boundBoxFunc' throws error

### ISSUE THREAD

**Issue by notchris** [konvajs/vue-konva#35]
When setting the configuration of a transformer, I tried to set the boundBoxFunc and it thew the following error:

> Cannot read property 'rotation' of undefined
    at Konva.Transformer._fitNodeInto

I set the config on the Transformer component.
`<v-transformer :config="transformConfig" ref="transformer" />`

And the config is setup like this:
`transformConfig: {
			ignoreStroke: true,
			rotateEnabled: false,
			boundBoxFunc: function () {
				console.log('test')
			}
}`

Thanks for your help!

**Comment by notchris** [comment]
As a side note, I'm trying to snap the size of a rect to a grid. I attempted this on the '@transform' event directly on the rect, but updating the size with .setAttr() or .size(size obj) did not update the rect. I was able to update the rect width using .width(val) and then calling .getLayer().draw() on the rect, but this only worked when the size provided was larger than the current rect size. It also would not update if I called .height() directly after calling .width() then updating.
